Sources/FingerprintJS/Fingerprints/DeviceInfoTreeProvider.swift

```diff
@@ -1,43 +1,56 @@
 protocol DeviceInfoTreeProvider {
     func buildTree(_ configuration: Configuration) -> DeviceInfoItem
 
-    typealias VersionedInfoItem = (item: DeviceInfoItem, versions: [FingerprintJSVersion])
+    typealias AnnotatedInfoItem = (
+        item: DeviceInfoItem,
+        stabilityLevel: FingerprintStabilityLevel,
+        versions: [FingerprintJSVersion]
+    )
 
-    var versionedItems: [VersionedInfoItem] { get }
+    var annotatedItems: [AnnotatedInfoItem] { get }
 }
 
 extension DeviceInfoTreeProvider {
-    func itemsForVersion(_ version: FingerprintJSVersion) -> [DeviceInfoItem] {
-        return
-            versionedItems
-            .filter { $0.versions.contains(version) }
-            .map { $0.item }
+    fileprivate func itemsMatching(
+        stabilityLevel: FingerprintStabilityLevel,
+        version: FingerprintJSVersion
+    ) -> [DeviceInfoItem] {
+        annotatedItems
+            .filter { item in
```

Comment:
Maybe name this `annotatedItem` so that it's apparent it's a different item than the one below in the keyPath.

Reply:
Okay, changed.